src/pages/billing/Project.js

```diff
@@ -568,6 +580,34 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
       ])
     ]),
     [spendReportKey]: div({ style: { display: 'grid', rowGap: '0.5rem' } }, [
+      !!errorMessage && div({
+        style: {
+          backgroundColor: colors.danger(0.15), borderRadius: '4px',
+          boxShadow: '0 0 4px 0 rgba(0,0,0,0.5)', display: 'flex',
+          padding: '1rem', margin: '1rem 0 0'
+        }
+      }, [
+        div({ style: { display: 'flex' }, role: 'alert' },
```

Comment:
Good call on the alert role. However, having it at the top level causes everything to be read out as you expand/collapse the details button. I think it would be sufficient to put the `alert` role just on the `div` that has the shortened message-- so lines 601 and 608.